sql-plugin/src/main/scala/com/nvidia/spark/rapids/HostColumnarToGpu.scala

```diff
@@ -117,6 +117,23 @@ object HostColumnarToGpu {
         for (i <- 0 until rows) {
           b.appendUTF8String(cv.getUTF8String(i).getBytes)
         }
+      case (dt, nullable) if dt.isDecimalType =>
+        // TODO: cuDF should support appending unscaledValue for decimal
```

Comment:
Aren't we doing that for the row to columnar transition?

Reply:
Oh, it is a mistake. I've fixed it.